lib/ui/pointer.dart

```diff
@@ -228,7 +258,9 @@ class PointerData {
              'radiusMax: $radiusMax, '
              'orientation: $orientation, '
              'tilt: $tilt, '
-             'platformData: $platformData'
+             'platformData: $platformData, '
+             'scrollDeltaX: $scrollDeltaX, '
```

Comment:
what is physicalX set to when scrollDeltaX is non null / nonzero?

Reply:
hm, i see, that's where the point is.

Will any future signals involve more than two linear dimensions?

Reply:
I would expect swipe to use these fields; pinch zoom and rotate would need different fields.